app/cdap/components/WranglerGrid/SelectColumnPanel/DataTable/CheckboxInput/index.tsx

```diff
@@ -0,0 +1,58 @@
+/*
```

Comment:
plz read the whole comment carefully before u make this change

the component folder name should be the title case 
plz connect with me to resolve this
the title case-sensitive changes are effective git and we have to change it carefully
use git mv for this

e.g.
```
git mv current-file-path/Foldername current-file-path/temp
git mv temp current-file-path/temp current-file-path/FolderName
```

plz read carefully before you make this change

Reply:
Done